Localization/README-ES.md

```diff
@@ -0,0 +1,157 @@
+# EXILED - EXtended In-runtime Library for External Development
+
+![EXILED CI](https://github.com/galaxy119/EXILED/workflows/EXILED%20CI/badge.svg?branch=2.0.0)
+<a href="https://github.com/Exiled-Team/EXILED/releases">
+  <img src="https://img.shields.io/github/release/Exiled-Team/EXILED/all.svg?style=flat" alt="GitHub Releases">
+</a>
+![Github All Downloads](https://img.shields.io/github/downloads/galaxy119/EXILED/total.svg?style=flat)
+![Github Commits](https://img.shields.io/github/commit-activity/w/Exiled-Team/EXILED/dev)
+<a href="https://discord.gg/PyUkWTg">
+  <img src="https://img.shields.io/discord/656673194693885975?logo=discord" alt="Chat on Discord">
+</a>
+
+
+EXILED es una plataforma de desarrollo de plugins para servidores de SCP: Secret Laboratory. Ofrece un sistema de eventos para desarrolladores y poder modificar o cambiar código del juego, o implementar sus propias funciones.
+Todos los eventos de EXILED están hechos con Harmony, lo que significa que no es necesario editar el código del juego/servidor para funcionar, esto ofrece dos beneficios únicos.
+
+ - Primeramente, toda la plataforma de desarrollo se puede publicar y distribuír, lo cual permite que los desarrolladores entiendan mejor *como funciona*, al igual que poder sugerir ideas o cambiar algo.
+ - Segundo, ya que todo el código relacionado con EXILED no está integrado en el código del servidor, cosas como actualizaciones pequeñas tendrán un pequeño (si acaso) efecto. Esto permite que siga siendo compatible con actualizaciones futuras, también agiliza el proceso de actualización cuando de verdad *es* necesario.
+
+# READMEs Traducidas
+- [中文](https://github.com/Exiled-Team/EXILED/blob/master/Localization/README-中文.md)
+- [English](https://github.com/Exiled-Team/EXILED/blob/master/README.md)

```

Comment:
I think spanish translated README needn't link to others translations

Reply:
Makes sense, I just duplicated the other one, It adds more consistency between READMEs, it also shouldnt bother anyone